perf/libunwind.py

```diff
@@ -48,18 +48,29 @@ def setUp(self):
                             "the test to be run" % package)
 
         tarball = self.fetch_asset('vanilla_pathscale.zip', locations=[
-            'https://github.com/pathscale/libunwind/archive/'
-            'vanilla_pathscale.zip'], expire='7d')
+            'https://github.com/libunwind/libunwind/archive/'
+            'master.zip'], expire='7d')
         archive.extract(tarball, self.workdir)
-        self.sourcedir = os.path.join(self.workdir, 'libunwind-vanilla_pathscale')
+        self.sourcedir = os.path.join(self.workdir, 'libunwind-master')
         os.chdir(self.sourcedir)
         process.run('./autogen.sh', shell=True)
         '''
         For configure options on different architecture please refer
-        https://github.com/pathscale/libunwind
+        https://github.com/libunwind/libunwind
         '''
+        configure_option = ''
+        if cpu.get_vendor() == 'intel':
+            configure_option = 'CC=icc CFLAGS="-g -O3 -ip" CXX=icc CCAS=gcc ' \
+                               'CCASFLAGS=-g LDFLAGS="-L$PWD/src/.libs"'
+        elif cpu.get_vendor() == 'ibm':
+            configure_option = 'CFLAGS="-g -O2 -m64" CXXFLAGS="-g -O2 -m64"'
+        else:
+            self.cancel(
```

Comment:
Why are we cancelling before reading yaml parameter ?

Reply:
ah right let me see how I can tweak this